### PR TITLE
musescore: update to 3.6.2.

### PR DESCRIPTION
**Pull request by tibequadorian**
Change maintainer (https://github.com/void-linux/void-packages/issues/29061)

I'll see if I can get cross builds working...

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by tibequadorian** [comment]
is it right that package contains /usr/bin/QtWebEngineProcess?

**Comment by tibequadorian** [comment]
Cross builds now work!

With qt5-webengine they only work with the patch I've included, only for little-endian build target and only if word size of host and target matches.
Luckily it's a really unnecessecary dependency on musescore, therefore I've disabled it by default. Patch is still there if someone wants to enable it.

Still needs testing because I've only tested it on x86_64-glibc.

**Comment by tibequadorian** [comment]
Wanna take a look, @ericonr ? :D

**Comment by tibequadorian** [comment]
update: removed the qt5-webengine option completely, there's no reason to keep it tbh

File: srcpkgs/musescore/template [hunk 1]
```diff
@@ -1,38 +1,33 @@
...
+maintainer="tibequadorian <tibequadorian@posteo.de>"
 license="GPL-2.0-or-later"
 homepage="https://musescore.org/"
-distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz"
-checksum=75b8e783edca9d23718770b3f715d95b88f64545e0372f4ccba58ab073cc7068
-nocross="genManual: cannot execute binary file: Exec format error"
+_soundfont_url="https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General"
+distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz
+ ${_soundfont_url}/MuseScore_General.sf3
+ ${_soundfont_url}/MuseScore_General_License.md"
```
Comment (ericonr) [inline srcpkgs/musescore/template:24]: Are there no versioned variants of this file? If upstream changes them, the template will just stop building :/

**Review by ericonr — COMMENTED** [review]
Can you explain in the commit message why the qtwebengine option isn't necessary?

File: srcpkgs/musescore/template (hunk 1 above)
Comment (tibequadorian) [reply srcpkgs/musescore/template:24]: I don't think so :/ These are the same that CMakeLists.txt uses to download ... at least they're checksummed now, remember https://github.com/void-linux/void-packages/pull/27451 ?

**Comment by tibequadorian** [comment]
> Can you explain in the commit message why the qtwebengine option isn't necessary?

Yeah I mean it just provides a small bar in the start center that links to several youtube videos about the software... nothing anyone will even notice

File: srcpkgs/musescore/template (hunk 1 above)
Comment (tibequadorian) [reply srcpkgs/musescore/template:24]: so if upstream changes them, they would have to update too

**Pull request merged**